source/docs/prismic/expose-content.md

```diff
@@ -0,0 +1,368 @@
+
+* `loadSingle(typeIdentifier[, options])` returns [an object of type `Content`](https://gitlab.com/front-commerce/front-commerce-prismic/-/blob/0.5.0/prismic/server/modules/prismic/core/loaders/Content.js) or throws an error if no such Content exists.
+* `loadByUID(typeIdentifier, uid[, options])` returns a `Content` representing a Prismic Content of [the corresponding type](https://prismic.io/docs/core-concepts/custom-types) and having [an UID field](https://prismic.io/docs/core-concepts/uid) with the given value. If such Content does not exist, it throws an error.
+* `loadList(query[, options])` returns [a `ContentList`](https://gitlab.com/front-commerce/front-commerce-prismic/-/blob/0.5.0/prismic/server/modules/prismic/core/loaders/ContentList.js) matching the query. `query` must be an instance of [`ListQuery`](https://gitlab.com/front-commerce/front-commerce-prismic/-/blob/0.5.0/prismic/server/modules/prismic/core/loaders/ListQuery.js), it provides a way to filter, sort and paginate Prismic Content.
+
+### Field Transformers
+
+By default, the `Content` objects returned by those methods directly expose the fields defined in the corresponding Prismic Custom Type. For instance if you are retrieving a Content of type `homepage` and [this Custom Type is single](https://prismic.io/docs/core-concepts/custom-types#difference-between-single-and-repeatable) and defines a `lastUpdate` field, you can write something like:
+
+```js
+const homepage = await loaders.Prismic.loadSingle("homepage");
+// homepage is an instance of Content
+// the lastUpdate field can be used directly,
+// it's value is the field value returned by the Prismic API
+console.log("Last update of my homepage", homepage.lastUpdate);
+```
+
+For most Field Types, the field value can directly be used. However for some of them, the field value is not very handy to expose the data in the Graph or even requires a deep transformation. To make that operation easier, the Prismic loader also exposes some Field Transformers to transform field values. In the previous example, if the `lastUpdate` field is [a Date Field](https://prismic.io/docs/core-concepts/date), `homepage.lastUpdate` will be a string. If instead you want to have a JavaScript `Date` instance as the field value, you can configure `loadSingle` to transform it:
+
+```js
+const { DateTransformer } = loaders.Prismic.transformers;
+const homepage = await loaders.Prismic.loadSingle("homepage", {
+  fieldTransformers: {
+    lastUpdate: new DateTransformer(),
+  }
+});
+// homepage.lastUpdate is now a Date object
+console.log("Last update of my homepage", homepage.lastUpdate);
+```
+
+The complete list of available Transformers can be found under [the `transformers` directory in Prismic module](https://gitlab.com/front-commerce/front-commerce-prismic/-/tree/0.5.0/prismic/server/modules/prismic/core/loaders/transformers). We will use some of them in the following examples.
+
```

Comment:
It could be good to add a paragraph highlighting the fact that all loader methods allowing to read data from Prismic accepts a `fieldTransformers` option to convert data.

Reply:
fixed in 122b8c5